### PR TITLE
Prac 07 feedback

### PR DESCRIPTION
**Pull request by a-rosa**
@AdriantoHartono
@nora-zyy
@JevanChandra

### CODE REVIEW

File: prac_07/programming_language.py
Comment (JevanChandra) [inline prac_07/programming_language.py:1]: Great Code!

File: prac_07/myguitars.py
Comment (JevanChandra) [inline prac_07/myguitars.py:1]: Nice Code!

File: prac_06/guitar.py
Comment (JevanChandra) [inline prac_06/guitar.py:1]: Great Job!

File: prac_07/language_file_reader.py
Comment (JevanChandra) [inline prac_07/language_file_reader.py:1]: Great way to solve this section!

File: prac_07/project_mangement.py
Comment (JevanChandra) [inline prac_07/project_mangement.py:1]: Good Job solving! This question is quite hard

File: prac_07/project.py
Comment (JevanChandra) [inline prac_07/project.py:1]: Nice work

File: prac_07/project_mangement.py
Comment (nora-zyy) [inline prac_07/project_mangement.py:1]: The thinking is clear and the code is very logical and easy to understand.

File: README.md
Comment (AdriantoHartono) [inline README.md:1]: Great 👌

File: prac_07/guitars.csv
Comment (AdriantoHartono) [inline prac_07/guitars.csv:1]: Nice🙏

File: prac_07/language_file_reader.py
Comment (AdriantoHartono) [inline prac_07/language_file_reader.py:1]: This code has been well written👍

File: prac_07/languages.csv
Comment (AdriantoHartono) [inline prac_07/languages.csv:1]: Nice

File: prac_07/myguitars.py
Comment (AdriantoHartono) [inline prac_07/myguitars.py:1]: Great code 👍

File: prac_07/programming_language.py
Comment (AdriantoHartono) [inline prac_07/programming_language.py:1]: Nice code👌

File: prac_07/project.py
Comment (AdriantoHartono) [inline prac_07/project.py:1]: Great 🙏

File: prac_07/project_mangement.py
Comment (AdriantoHartono) [inline prac_07/project_mangement.py:1]: The code has been well organized in a function, which make it much tidier🙏

**Pull request merged**